styles/Home.module.css

```diff
@@ -50,6 +50,8 @@
 .hide{
   display: none;
 }
+.smnav{
```

Comment:
<img width="1201" alt="image" src="https://github.com/WebSorcery/testcase-generator/assets/32160084/66d3bdd8-a2c4-4c1e-901c-b24a76d0a31f">

@singhcoder694 Please look into this too

Reply:
<img width="177" alt="Screenshot 2023-12-23 100036" src="https://github.com/WebSorcery/testcase-generator/assets/114601400/d0401d1d-dddd-45df-9a6d-85069276a3cf">
I did this to manage two navbar problem. But this may have changed due to last PR you merged for UI/UX design, at my side it is like the image above showing no error.

Reply:
Okay, please push the fix.

Reply:
Ok